contracts/modules/NftTransfer.sol

```diff
@@ -0,0 +1,136 @@
+pragma solidity ^0.5.4;
+
+import "./common/BaseModule.sol";
+import "./common/RelayerModule.sol";
+import "../storage/GuardianStorage.sol";
+
+/**
+ * @title NftTransfer
+ * @dev Module to transfer NFTs (ERC721),
+ * @author Olivier VDB - <olivier@argent.xyz>
+ */
+contract NftTransfer is BaseModule, RelayerModule {
+
+    bytes32 constant NAME = "NftTransfer";
+
+    // Equals to `bytes4(keccak256("onERC721Received(address,address,uint256,bytes)"))`
+    bytes4 private constant ERC721_RECEIVED = 0x150b7a02;
+
+    // The Guardian storage 
+    GuardianStorage public guardianStorage;
+
+    // *************** Events *************************** //
+
+    event NonFungibleTransfer(address indexed wallet, address indexed nftContract, uint256 indexed tokenId, address to, bytes data);    
+
+    // *************** Modifiers *************************** //
+
+    /**
+     * @dev Throws if the caller is not the owner or an authorised module.
+     */
+    modifier onlyOwnerOrModule(BaseWallet _wallet) {
```

Comment:
Why do we want another module to call this one? I'm afraid it may open the door to attacks so we need to make sure we do it for a good reason.

Reply:
Yes, you are right. And likewise, I think there is no reason for `TokenTransfer.transferToken()` to have this modifier either (maybe it is a remnant from when `DappManager.callContract()` was calling `TokenTransfer.transferToken()` ?). Shall I also change it for `TokenTransfer` so the change is there for the next time we redeploy `TokenTransfer`?

Reply:
Yes.